src/overlays/actors/ovl_En_Ex_Item/z_en_ex_item.c

```diff
@@ -488,10 +488,10 @@ void EnExItem_DrawHeartPiece(EnExItem* this, GlobalContext* globalCtx) {
 }
 
 void EnExItem_DrawMagic(EnExItem* this, GlobalContext* globalCtx, s16 magicIndex) {
-    static s16 sgiDrawIds[] = { GID_DINS_FIRE, GID_FARORES_WIND, GID_NAYRUS_LOVE };
+    static s16 giDrawIds[] = { GID_DINS_FIRE, GID_FARORES_WIND, GID_NAYRUS_LOVE };
```

Comment:
The previous name wasn't good either so good catch, but technically this should have the `s` prefix since it's static.
It's something we need to fix in general because it wasn't always applied consistently over time, so I'll probably do a full cleanup soon if no one gets to it before me.

Reply:
we havent been doing it for in function static vars on purpose though

Reply:
in-function static should have the `s` prefix too? that's more the exception than the norm overall :sweat_smile:

Reply:
In-function static variables are still very different from local variables so I think it's important to have the prefix. Also, whether a static variable is inside or outside function doesn't change anything besides its scope, so to me I doesn't really make sense to name it differently.

(Relevant discussion we had on discord about this, where I brought up why naming without the prefix could be very misleading: https://discord.com/channels/688807550715560050/688851317593997489/887738378525749268)

Reply:
I don't really care about prefixing in-function static with `s` (I do think it's slightly better)
But regardless of going with that or not, what should this PR do about it? (I think nothing, there's enough stuff in there already)

Reply:
agreed on doing nothing btw, and roman said he'll clean up the variable names at a later time

Im still a bit confused on the whole `s` thing for in function static variables, we made it a point in reviews thus far to remove them for in function static. But I do also understand the reasoning of keeping them to make them stand out from normal variables.

Either way its a conversation that needs to happen separately outside of this pr